pkg/events/derive/derive.go

```diff
@@ -2,14 +2,39 @@ package derive
 
 import (
 	"fmt"
+	"sync"
 
+	"github.com/aquasecurity/tracee/pkg/counter"
 	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/events/parse"
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
 // Use as static variable for testability reasons
 var getEventDefFunc = events.Definitions.Get
 
+// Context is a struct used to store the triggering events' context
+type Context struct {
```

Comment:
I think this mechanism can get it's own file.

Reply:
Agreed, this will be included in the `triggeredEvents` package as mentioned in #2055

Reply:
We can move it already inside the current package, then just move it later.